setup.py

```diff
@@ -171,9 +172,6 @@
             "data/*.js",
             "data/*.png",
             "data/colormaps/*.cpt",
-            "data/input_files/*.txt",
-            "data/input_files/*.template",
-            "data/input_files/*.md",
```

Comment:
Only question I had was if there was anything relying on the templates being included as package data?

Reply:
The code is not relying on these.

But it's was used in the example dataset page: https://github.com/insarlab/MintPy/blob/main/docs/demo_dataset.md, which I forgot to update.

Reply:
Updated.